src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.cpp

```diff
@@ -219,7 +223,11 @@ void ov::npuw::LLMInferRequest::infer_prefill(ov::SoPtr<ov::ITensor> input_ids,
         static_cast<uint32_t>(input_ids->get_shape()[INPUT_IDS_SEQ_LEN_DIM]);
     m_need_copy_kvcache = true;
 
-    m_logits = m_prefill_request->get_tensor(m_prefill_out_ports.at("logits"));
+    if (m_tail_mm_request_opt.has_value()) {
+        infer_tail_mm(m_prefill_request->get_tensor(m_prefill_out_ports.at("output_embed")));
```

Comment:
these string literals should be CONST-ed somewhere I believe.

Reply:
As another names, that are official: `input_ids`, `position_ids`, 'attention_mask` and so on are preserved as string literals, could I leave this name as string literal also for consistency?

Reply:
those should be literals too.

Reply:
Thanks, fixed!